src/bin/deku_node.ml

```diff
@@ -591,18 +610,33 @@ let info_add_trusted_validator =
 let add_trusted_validator node_folder address =
   let open Network in
   let%await identity = read_identity ~node_folder in
+  let%await state = Node_state.get_initial_state ~folder:node_folder ~minimum_block_delay:0. ~pollinate_context:Node.State.Client in 
   let payload =
-    let open Trusted_validators_membership_change in
     { address; action = Add } in
   let payload_json_str =
     payload
-    |> Trusted_validators_membership_change.payload_to_yojson
+    |> validators_payload_to_yojson
     |> Yojson.Safe.to_string in
+
   let payload_hash = BLAKE2B.hash payload_json_str in
   let signature = Signature.sign ~key:identity.secret payload_hash in
-  let%await () =
-    Network.request_trusted_validator_membership { signature; payload }
-      identity.uri in
+  let%await interop_context = interop_context node_folder in
+  let%await validator_uris = validator_uris ~interop_context in
+  match validator_uris with
+  | Error err -> Lwt.return (`Error (false, err))
+  | Ok validator_uris ->
+    let open Network.Pollinate_utils in
+    let validator_uris = List.map snd validator_uris |> List.somes in
+    let recipients = List.map uri_to_pollinate validator_uris in
```

Comment:
To continue previous comment: you got this functionality wrong. The goal of this command line action is not to add a validator to a chain, it's to tell a given _node_ that it should accept the addition or removal, through a `Consensus_operation`, of a validator. You tell just one node: the one for which you have the node folder.

Hence, no list of URIs to send messages to.